sdk_container/src/third_party/portage-stable/acct-user/pcscd/pcscd-0-r2.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="A user for pcsc-lite"
+ACCT_USER_ID=47
+#ACCT_USER_GROUPS=( pcscd openct usb )
```

Comment:
The special handling is needed when the group ID specified in the ebuild is different from one that is already used in Flatcar. In case of `usb` group, the IDs are the same (85), so not sure what's going on here at first. Where's the error message?